app/models/manageiq/providers/ansible_tower/shared/inventory/parser/automation_manager.rb

```diff
@@ -53,7 +53,7 @@ def configuration_script_sources
       inventory_object.description = project.description
       inventory_object.name = project.name
       # checking project.credential due to https://github.com/ansible/ansible_tower_client_ruby/issues/68
-      inventory_object.authentication = persister.credentials.lazy_find(project.credential_id.to_s) if project.credential
+      inventory_object.authentication = persister.credentials.lazy_find(project.credential_id.to_s) if project.credential_id
```

Comment:
is the issue with Ansible Tower ruby client resolved or behavior changed? this check on `project.credential_id` used to be failing in some cases